tools/atip/README.md

```diff
@@ -401,10 +401,11 @@ def i_visit_url(context, url):
 @step(u'I edit view "{params_kw}" to input "{text}"')
 @step(u'I edit index {n:d} EditText to input "{text}"')
 @step(u'I compare view "{params_kw}" info "{info_name}" with "{except_result}"')
```

Comment:
The view "{params_kw}" info "{info_name}" should be "{except_result}"
would be better.